lib/jekyll-relative-links/generator.rb

```diff
@@ -64,11 +64,15 @@ def markdown_converter
     def url_for_path(path)
       extension = File.extname(path)
       return unless markdown_extension?(extension)
-
-      page = site.pages.find { |p| p.path == path }
+      page = find_target(path)
       relative_url(page.url) if page
     end
 
+    def find_target(path)
+      site.pages.find { |p| p.path == path } ||
+        site.docs_to_write.find { |p| p.relative_path == path }
+    end
```

Comment:
Would it make sense to have one method that returns `(site.pages + site.docs_to_write)` and then here, like on line 29, we can simply iterate through the pages/docs once looking for `page.relative_path`?